vcd/config.go

```diff
@@ -90,6 +91,79 @@ func debugPrintf(format string, args ...interface{}) {
 	}
 }
 
+// This is a global MutexKV for all resources
+var vcdMutexKV = mutexkv.NewMutexKV()
+
+func (cli *VCDClient) lockVapp(d *schema.ResourceData) {
+	vappName := d.Get("name").(string)
+	if vappName == "" {
+		panic("vapp name isn't found")
```

Comment:
Please change all occurrences also in the other functions below
"vapp name isn't found" -> "vApp name not found"

Reply:
Fixed